apps/style/craft/style.scss

```diff
@@ -116,6 +116,9 @@ body {
 
 #belowVisualization p {
   color: white;
+  & .legacy-share-view {
+    display: none;
+  }
```

Comment:
Is this necessary, since you hide all of `#belowVisualization` when using `legacy-share-view` in common.scss, above?